### ISSUE TITLE
Switch from forum scrapper to Floatplane Website API

### ISSUE THREAD

**Issue by alex73630** [alex73630/Floatplane-Downloader#12]
As AJJaxNet (on the LTT Discord) confirmed me, the download feature on the new website should be kept for the final release and as there is already some kind of undocumented API to handle videos. 

A new and full featured client, based on the previous work that I started with the v0.2.0, supporting multi channels (I may add some kind of regex filters to separate series/sub-channels, like "TechQuickie" from "Linus Tech Tips", if the API doesn't provide those infos)

I may add some kind of API support for Plex, to be able to manage the Floatplane library from the Floatplane-Downloader client. This should help to fix the duplicate video title if there is 2 new releases the same day. A custom agent could also be a great idea, so it could grab infos directly from the app and would only need a one time setup (but still require "advanced knowledge").

tldr: APP IS NO MORE IN "NOT MAINTAINED" STATE, NEW AND WAY BETTER VERSION INCOMING ! #v3Hype !
